### PR TITLE
Tbi/ui5version in yaml/17765

### PR DESCRIPTION
**Pull request by unseen1980**
For TBI https://github.com/SAP/open-ux-tools/issues/333

- Remove version from ui5-config
- Update TS types
- Update tests/snapshots

### CODE REVIEW

**Review by IainSAP — COMMENTED** [review]
@unseen1980 I think we can remove this now https://github.com/SAP/open-ux-tools/blob/main/packages/ui5-application-writer/src/index.ts#L38 ?

**Review by IainSAP — APPROVED** [review]
OK for me, not sure we need 2 change sets? Removal of version from ui5-config api doesn't result in an API change for ui5-application-writer. Thx @unseen1980

File: packages/ui5-config/src/types.ts [hunk 1]
```diff
@@ -15,7 +15,6 @@ export interface ProxyUIConfig {
     [key: string]: unknown | undefined;
     path?: string[];
     url?: string;
-    version?: string;
```
Comment (zdravko-georgiev) [inline packages/ui5-config/src/types.ts:18]: The `version` is still a valid property. It just doesn't need to be set by default. I would keep it in the type's definition.

**Review by zdravko-georgiev — CHANGES_REQUESTED** [review]

File: packages/ui5-config/src/types.ts (hunk 1 above)
Comment (IainSAP) [reply packages/ui5-config/src/types.ts:18]: Isnt it redundant as its not referenced anymore?

**Review by zdravko-georgiev — APPROVED** [review]
LGTM

**Review by Klaus-Keller — APPROVED** [review]
Did a review of code, no build and test. However, out of curiosity I've tried to remove the `version` property from `ui5.yaml` und started the preview using an older `@sap/ux-ui5-middleware` (1.4.1) and it still worked. This situation should not happen as only new generated projects have the `version` removed, but you never know.

**Pull request merged**